jupyter_server_ydoc/handlers.py

```diff
@@ -233,7 +244,10 @@ async def maybe_load_document(self):
 
     async def send(self, message):
         # needed to be compatible with WebsocketServer (websocket.send)
-        self.write_message(message, binary=True)
+        try:
+            self.write_message(message, binary=True)
+        except BaseException:
+            pass
```

Comment:
It would be interesting to log for debugging, isn't it?

```suggestion
        except Exception as e:
            self.log.debug("Failed to write message", exc_info=e)
```